chore(website_linter): Prioritize config file example snippet in doc_page.

connorshea opened the pull request:
In almost all cases, users should be configuring rules individually via the config file. If they are not.... I don't want to think about that. So anyway, show that option first.

camc314 commented:
### Merge activity

* **Dec 7, 12:10 PM UTC**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.com/merges?org=oxc-project&repo=oxc) once it meets the requirements.
* **Dec 7, 12:11 PM UTC**: `camc314` added this pull request to the [Graphite merge queue](https://app.graphite.com/merges?org=oxc-project&repo=oxc).

connorshea committed:
chore(website_linter): Prioritize config file example snippet in doc_page. (#16566)

In almost all cases, users should be configuring rules individually via the config file. If they are not.... I don't want to think about that. So anyway, show that option first.
